CONTRIBUTING.md

```diff
@@ -74,9 +74,12 @@ You can also use these NPM tasks (see `npm run` for the full list):
 
 ## Develop
 
-### Code guidelines
+### Guidelines
 
-See [CODE_GUIDELINES.md](./docs/CODE_GUIDELINES.md) for coding conventions.
+- Project practices and patterns: [CODE_GUIDELINES.md](./docs/CODE_GUIDELINES.md)
+- [VS Code UI guide](https://code.visualstudio.com/docs/getstarted/userinterface)
```

Comment:
I feel like this one could be moved to the CODE_GUIDELINES with some best practices? UI guide doesn't mean much unless we say what we'll do with it.

Reply:
Arguably this should be an entirely different document about UI.

Reply:
Added it there for now.

I am wondering if we will need  our own UI_GUIDELINES.md soon.